Makefile

```diff
@@ -47,27 +35,27 @@ build: build-linux-amd64
 build-tmp: bin # build tmp exec to perform adlr steps
 	@$(GOBUILD) -o $(BIN)/tmp ./$(ADLR_MAIN)
 
-build-linux-amd64: licenselock version
+build-linux-amd64: version
 	@$(SCRIPTS)/build.sh \
 	adlr linux amd64 ./$(ADLR_MAIN) ./$(BIN)
 
-buildlist:
+buildlist: tidy
 	@$(GOLIST) -m -json all > $(BUILDLIST)
 
-licenselock: build-tmp buildlist
```

Comment:
Again, the `license.lock` file is replaced by the output of `adlr-cli license verify` (e.g., `verified-licenses.json`). I could technically update this make command to generate a file called `license.lock` using the three `adlr-cli` license commands, but that is misleading as it would most likely only contain a subset of the licenses that need to be located, identified, and verified. Often times, the info in `buildlist.json` is not sufficient for locating all licenses, and/or the text in the found license files can't accurately be used to identify the type of license. Meaning, there is a fair bit of manual work that needs to be done to ensure each `adlr-cli license` subcommand has the information necessary for checking _all_ of our dependencies